src/ext/Util/ca/TouchFile.cpp

```diff
@@ -23,6 +23,7 @@ static HRESULT SetExistingFileModifiedTime(
     )
 {
     HRESULT hr = S_OK;
+#ifndef _WIN64
```

Comment:
ifndef/else bugs me but I realize it's just a pet peeve.

Reply:
(as opposed to ifdef/else)

Reply:
Agreed but all of the other WiX CA code that deals with WOW64 redirection uses the same `#ifndef _WIN64` pattern. So does the rest of the code in this file. So,I thought it would be even more confusing to have this be the one place it was not `#ifndef`.

Reply:
Not all of it; RemoveFolderEx does it my way. :) But then it means it switches from ifdef to ifndef when there's no else clause. So, 50/50.